src/m_menu.c

```diff
@@ -4124,8 +4124,16 @@ void M_ResetTimeScale(void)
   }
 }
 
+static void M_UpdateDirectVerticalAimingItem(void)
+{
+  DISABLE_ITEM(!mouselook && !padlook, gen_settings3[gen3_verticalaim]);
```

Comment:
Shouldn't we check for `strictmode` too? I'm not sure if that's handled automatically by the `S_STRICT` flag.

Reply:
It's already checked by `S_STRICT` flag.